samples/kubernetes/end2end/docs/dashboard.md

```diff
@@ -0,0 +1,22 @@
+## What's in the Weblogic Dashboard
+### Overview
+The WebLogic Dashboard helps you know what's going on inside WebLogic servers, how much traffic they're getting, how they're performing, any errors etc.
+
+On top of the Weblogic dashboard, there are some filters which allows you to filter out metrics of different domain, different cluster and different server(s). Another filter is `Top N` whose candidates values are 3, 5, 7 and 10. The value will be applied to some panels to only show the top N elements to avoid too many elements displayed.  
```

Comment:
Replace this paragraph with:
On top left hand corner of the Weblogic Server dashboard page, there are some filters which allows you to filter out metrics based on different domains, different clusters, and different servers. Another filter is the `Top N` whose candidates values are 3, 5, 7 and 10. The value will be applied to some panels to only show the top N elements to avoid too many elements displayed.